tests/jsunit/field_variable_test.js

```diff
@@ -146,7 +148,8 @@ function test_fieldVariable_dropdownVariableAndTypeDoesNotExist() {
   var result_options = Blockly.FieldVariable.dropdownCreate.call(
```

Comment:
Done and titled:
function test_fieldVariable_getVariableTypes_undefinedVariableTypes()